packages/material-ui/src/ClickAwayListener/ClickAwayListener.tsx

```diff
@@ -26,7 +26,7 @@ export interface ClickAwayListenerProps {
   /**
    * The wrapped element.
    */
-  children: React.ReactElement;
+  children?: React.ReactElement;
```

Comment:
We do require a single child. This prop is not optional.

Reply:
How to solve the Typescript issue then?

Reply:
@tonoli Do you have a reproduction? You can start from https://codesandbox.io/s/clickaway-material-demo-forked-u9yrp?file=/demo.tsx.